iotilecore/iotile/core/hw/commands/rpc.py

```diff
@@ -1,98 +0,0 @@
-# This file is adapted from python code released by WellDone International
-# under the terms of the LGPLv3.  WellDone International's contact information is
-# info@welldone.org
-# http://welldone.org
-#
-# Modifications to this file from the original created at WellDone International
-# are copyright Arch Systems Inc.
-
-from builtins import str, int
```

Comment:
These two imports need to be ported over to `TileBusProxyObject` now.  They are provided by the python `future` module to backport python 3 unicode features to python 2.  It's important because the check on line 101:

https://github.com/iotile/coretools/pull/594/files#diff-7a7c0562805a6437e11a53d9b89d8fc0R101

will be incorrect without it since on python 2 `str` and `bytes` are the same so it will try to `utf-8` encode binary data, which is why this test is failing:
https://travis-ci.org/iotile/coretools/jobs/460641356#L603

Reply:
@mattrunchey